app/Console/Commands/LdapSync.php

```diff
@@ -253,8 +253,8 @@ public function handle()
 
                 if ($item['ldap_location_override'] == true) {
                     $user->location_id = $item['location_id'];
-                } elseif ((isset($location)) && (! empty($location))) {
```

Comment:
Should these changes be in here? They don't seem related to code quality.

Reply:
Fixed